R/angles.R

```diff
@@ -26,43 +23,118 @@ calculate.angles.radial <- function(v, tree, spread, total.angle) {
         num.children <- length(child.ids);
 
         if (num.children > 0) {
-            parent.angle <- angles[current.node.id];
+            parent.angle <- angles[tree$tip == current.node.id];
```

Comment:
I assume this is to support arbitrary node IDs? (As opposed to automatically generated, ordered node IDs that can be used to index directly)